apps/compact-log-viewer/compact-log-viewer.yml

```
@@ -0,0 +1,12 @@
+name: 'Compact Log Viewer'
+description: 'A cross platform tool to read & query JSON aka CLEF log files created by Serilog'
```

[Don't start the description with A or An.](https://github.com/electron/apps/blob/master/contributing.md#submission-guidelines)

```suggestion
description: 'Cross-platform tool to read & query JSON aka CLEF log files created by Serilog'
```